test/runtests.jl

```diff
@@ -1,7 +1,12 @@
 using Test
 using RSEinJulia
+using Aqua: Aqua
 
 @test_nowarn greet()
 
+@testset "Aqua.jl" begin
+    Aqua.test_all(2023-RSE_in_Julia)
```

Comment:
```suggestion
    Aqua.test_all(RSEinJulia)
```
This should be the module. Did you run tests locally?